Lesson Plans/Private/XC.9 Emergency Operations.md

```diff
@@ -23,42 +23,107 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
+
+##### Something is Wrong!
+DISCLAIMER: The POH has procedures for many of these situations, and you should follow them.  This bulleted list is intended to help you think through the list of emergencies in a logical way, debug the problem, and understand at a high level how you'll handle the situation.  **(POH)** means the POH has a procedure you should learn, with initial steps (in bold) that you need to memorize to execute quickly.
+
+- On the ground...
+	- Fire! - **(POH)** Cut off fuel to the engine; grab fire extinguisher; evacuate
+- During takeoff...
+	- Fire!
+		- **(POH)** Can you abort takeoff?  Abort takeoff.
+		- Else, take off, and land straight out
+	- Engine Failure!
+		- **(POH)** Abort takeoff, or forced landing straight ahead.^[[Cub Theatre: The-Not-That-Impossible Turn](https://www.youtube.com/watch?v=byBh1Qd6IgA) from [[Paul Bertorelli]] of [[AVweb]]]
+	- Airspeed isn't "live"
+		- Abort takeoff, or fly the pattern and land normally
+	- A door opens
+		- Takeoff normally.  Deal with it in the air, or land normally
+- In the air...
+	- Fire!  Where?
+		- Engine cowling?
+			- **(POH)** Cut off fuel flow; Emergency Descent Procedure; don't turn off electrical unless fire is electrical
+		- Inside the dashboard (probably electrical) or cabin?
+			- **(POH)** Master and stby switches off; deploy fire extinguisher
+				- Fire extinguished? - open windows to clear smoke; if required, follow procedure to re-power some items; Precautionary Landing
+				- Not extinguished? - Emergency Descent
+		- Wing?
+			- **(POH)** Turn wing lights and pitot heat off; sideslip to keep flames away; Precautionary landing
+	- Burning smell! (Possible electrical fire)
+		- **(POH)** Master and stby switches off; optionally follow procedure to re-power some items
+	- Engine failure!
+		- **(POH)** Best glide; aim for landing spot; attempt restart; Else execute Forced Landing
+	- Engine running poorly!
+		- Identify a landing spot and navigate toward it; Climb if possible, to buy time; Debug the problem, but be prepared for a Forced Landing
+		- Possible problems include:
+			- Carburetor icing - Add carb heat. [[How and Why Carburetor Ice Forms]]
+			- Bad mixture/throttle settings - Adjust
+			- Fuel pump failure - Turn on backup pump
+			- Fuel tank running dry - Switch tanks
+			- Ignition switch in 'A' or 'B' - Switch to 'both'
+	- Electrical failure
+		- The plane still flies; use backup radio; land ASAP
+	- Radio failure (you can't hear them, or they can't hear you)
+		- Squawk 7600; proceed to destination if safe; look for light signals
+	- Instruments don't seem to be reading correctly
+		- Are altimeter and VSI "stuck" even when you change altitude?
+			- **(POH) **Static source is blocked; try alternate static source
+		- Does airspeed vary with altitude, even when flying level?
+			- Pitot tube is blocked
+		- Are the vacuum-powered instruments dead? (Which ones are they in your plane?)
+			- Vacuum pump is dead
```

Comment:
I suppose somewhere in this debug flow should have things like "check the vacuum gauge" -- not sure where that would fit in the debug flow though